app/src/layouts/common/appSidebar/helpAndService/modal/requestSupportModal.jsx

```diff
@@ -100,7 +99,7 @@ const RequestSupport = ({ handleSubmit, initialize, invalid }) => {
   };
 
   return (
-    <ModalLayout
+    <Modal
       title={formatMessage(messages.requestService)}
       okButton={{
         text: formatMessage(messages.sendRequest),
```

Comment:
You also need to update buttons.
Please check corresponding PR - https://github.com/reportportal/service-ui/pull/3941.

Reply:
Done.

Reply:
Done.